app/routers/dogfinder.py

```diff
@@ -153,6 +163,7 @@ async def add_document(type: str = Form(...), breed: Optional[str] = Form(None),
             data_properties[UUID_FIELD] = generate_uuid5({"filename": dogDocument.filename})
             data_properties["document_embedding"] = dog_embedding
             data_properties[IS_FOUND_FIELD] = False
+            data_properties[DOG_ID_FIELD] = generate_dog_id()
```

Comment:
Lets move this to create_data_properties

Reply:
ok